matcher/service/openreview_interface.py

```diff
@@ -101,12 +104,38 @@ def get_all_edges(client, edge_invitation_id, logger=None):
             ))
     return all_edges
 
+class CacheHandler:
+    def __init__(self, host, port, cache_expiration):
+        self.redis_client = redis.Redis(host=host, port=port)
+        self.cache_expiration = cache_expiration
+        self.key_prefix = ''
+
+    def set_key_prefix(self, key_prefix):
+        self.key_prefix = key_prefix
+
+    def get_value(self, key):
+        serialized_value = self.redis_client.get(self.key_prefix + key)
+        if serialized_value:
+            return pickle.loads(serialized_value)
+        return False
+
+    def set_value(self, key, value):
+        serialized_value = pickle.dumps(value)
+        self.redis_client.setex(self.key_prefix + key, self.cache_expiration, serialized_value)
+
 class ConfigNoteInterface:
-    def __init__(self, client, config_note_id, logger=logging.getLogger(__name__), cache=None):
+    def __init__(self, client, config_note_id, cache_handler, logger=logging.getLogger(__name__)):
         self.client = client
         self.config_note_id = config_note_id
         self.logger = logger
-        self._cache = {} if not cache else cache
+        self._cache = {}
```

Comment:
Can we remove the var self._cache? I do not think it is not being used anywhere anymore.

Reply:
I think I need to work on this, I made too much changes in the interface and there are a lot of conflicts. I will ask your review.